content/blog/eng/careful-with-that-constant-developer.md

```diff
@@ -0,0 +1,458 @@
+---
+authors: ["amaccis"]
+comments: true
+date: "2021-02-07"
+draft: false
+share: true
+categories: [English, PHP, OOP]
+title: "Careful with that constant, developer"
+languageCode: "en-EN"
+type: "post"
+toc: true
+---
+
+# Introduction
+
+Imagine you need to use a fixed numeric value in your code, let's say 20000. It's what we call a *literal constant*, that is a value that will always remain the exact same in your code. 
+Now imagine you need to use that value more than once. Sooner or later you will read your code and not remember what 20000 was, it will happen for sure. But what if you assign the value to an immutable variable with a [meaningful name](https://en.wikipedia.org/wiki/Self-documenting_code)? Let's say something like this
```

Comment:
```suggestion
Now imagine you need to use that value more than once. Sooner or later you will read your code and not remember what 20000 was, it will happen for sure. But what if you assign the value to an immutable variable with a [meaningful name](https://en.wikipedia.org/wiki/Self-documenting_code)? Let's say something like this:
```

Reply:
I was re-reading the sentence and my review and I would suggest:

Now imagine you need to use that value more than once. Sooner or later you will read your code and **you will not** remember what 20000 was...